packages/core/parcel-bundler/src/assets/HTMLAsset.js

```diff
@@ -270,16 +270,15 @@ class HTMLAsset extends Asset {
       } else if (type === 'tag') {
         if (rendition.isMain) {
           node.content = rendition.value;
-        }
+          if (node.attrs) {
+            if (rendition.type === 'js') {
+              node.attrs.type = 'application/javascript';
```

Comment:
Wouldn't we just remove the attribute in that case since the default type is application/javascript? Same for the css case below

Reply:
I had the same question at some point specifically for the css part of it, you can see that conversation here, https://github.com/parcel-bundler/parcel/pull/2328#issuecomment-442561148.

I can definitely remove the attrs.type if that's the desire output of this PR.